src/Migrations/Utils/Constants.php

```diff
@@ -71,4 +83,19 @@ class Constants
         self::X => '/^(https?\:\/\/)?(www\.)?x\.com\/.+$/i',
     ];
     // phpcs:enable Generic.Files.LineLength.MaxExceeded
+
+    public const COVER_BLOCK_TYPES = [
+        'take-action' => [
```

Comment:
Can't we use other constants here? such as `COVER_TYPE_TAKE_ACTION`? 
Also, do we still need to `OLD_COVER_TYPES` constant? Seems we could use this one instead